docs/reference/troubleshoot.md

```diff
@@ -195,6 +197,70 @@ For example, for a `CloudSQLInstance` managed resource (`database.gcp.crossplane
 kubectl patch cloudsqlinstance my-db -p '{"metadata":{"finalizers": []}}' --type=merge
 ```
 
+## Handling Crossplane Package Dependency
+
+When use `crossplane.yaml` to define Crossplane package, you can specify packages that 
+the current package depends on including the version constraint using `spec.dependsOn`.
+
+Please make sure you provide the fully qualified address to the dependent package, but do
+not append the package version (i.e. the OCI image tag) after the package name. This may
+lead to the missing dependency error when Crossplane tries to install the dependency.
+
+When specify the version constraint, you should strictly follow the [semver spec](https://github.com/Masterminds/semver#basic-comparisons). Otherwise,
+it may not be able to find the appropriate version for the dependent package even it says
+the dependency is found. This may lead to the incompatible dependency error during the installation.
+
+Below is an example where a configuration package depends on a provider pulled from `crossplane/provider-aws`.
+It defines `">v0.16.0-0` as the version constraint which means all versions after `v0.16.0` including
+all prerelease versions, in the form of `-xyz` after the normal version string, will be considered when
+Crossplane tries to find the best match.
+
+```yaml
+apiVersion: meta.pkg.crossplane.io/v1
+kind: Configuration
+metadata:
+  name: test-configuration
+  annotations:
+    provider: aws
+spec:
+  crossplane:
+    version: ">=v1.0.0-0"
+  dependsOn:
+    - provider: crossplane/provider-aws
+      version: ">v0.16.0-0"
+```
+
+## Installing Crossplane Package
+
+After install Crossplane package, to verify the install results or troubleshoot any issue spot during the
+installation, there are a few things you can do.
+
+List all Crossplane resources available on your cluster:
+
+```console
+kubectl get crossplane
+```
+
+If you installed provider package, pay attention to the `Provider` and `ProviderRevision` resource. Especially
+the `INSTALLED` and `HEALTHY` column. They all need to be `TRUE`. Otherwise, there must be some errors occurred
+during the installation.
+
+If you installed configuration package, pay attention to the `Configuration` and `ConfigurationRevision` resource.
+Again, the `INSTALLED` and `HEALTHY` column for these resources need to be `TRUE`. Besides that, you should also 
+see the `CompositeResourceDefinition` and `Composition` resources included in this package are listed if the package 
+is installed successfully.
+
+When there are errors, you can run below command to check detailed information for the packages that are getting installed.
+
+```console
+kubectl get lock -o yaml
+```
+
+To inspect a particular package for troubleshooting, you can run `kubectl describe` against the corresponding resources,
```

Comment:
Looks like some of the lines in this PR are not wrapped at 80 chars, could you update to match the rest of the doc? :)